cdap-app-fabric/src/main/java/co/cask/cdap/metadata/MetadataAdmin.java

```diff
@@ -61,18 +66,43 @@ void addProperties(Id.NamespacedId entityId, Map<String, String> properties)
   Set<MetadataRecord> getMetadata(Id.NamespacedId entityId) throws NotFoundException;
```

Comment:
This function's doc need to updated to say that it will give metadata across scope